migrations/2_deploy_organization.js

```diff
@@ -6,9 +6,8 @@ module.exports = async (deployer) => {
 
   const gasAmount = Config.get("gasLimit_deployment");
   const network = Config.get("network");
-  console.log(`Deploying to ${network}, gasLimit: ${gasAmount}`);
-
   const founders = require("./founders.json").founders[network];
+  console.log(`Deploying to ${network}, gasLimit: ${gasAmount},  ${founders.length} founders`);
```

Comment:
please remove console.log

Reply:
how does it pass the eslint ?

Reply:
this is a script that logs lots of information on the console as a normal and useful part of its function.  I would prefer to leave this in.  Does that work for you?

Reply:
no.how the linter did not catch that ?